{
  LOGGER.error("Got exception in the channel to {}",_server,cause);
  closeOnError(ctx,cause);
}
